shared/src/question-generators/unionFind/quickFind/utilsQF.ts

```diff
@@ -0,0 +1,384 @@
+import { QuickFind } from "@shared/question-generators/unionFind/quickFind/quickFindAlgorithm"
+import { createArrayDisplayCodeBlock } from "@shared/utils/arrayDisplayCodeBlock.ts"
```

Comment:
`.ts` extension not needed.

Reply:
Will push the plugin for consistent imports tomorrow. Then this be resolved automatically 👍